[CTOR-1286]-Refresh hardware-storage-hitachi-hcp-snmp doc page (#3986)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-storage-hitachi-hcp-snmp.md

| bbu-status                                             | N/A   |
| *nodes*~node.space.usage.bytes                         | B     |
| *nodes*~node.space.free.bytes                          | B     |
| *nodes*~node.space.usage.percentage                    | %     |
| *nodes*~*temperatures*#node.sensor.temperature.celsius | C     |
| *nodes*~*fans*#node.sensor.fan.speed.rpm               | rpm   |
| *nodes*~*voltages*#node.sensor.voltage.volt            | V     |

</TabItem>
<TabItem value="Tenants" label="Tenants">

| Name                                    | Unit  |
|:----------------------------------------|:------|
| *tenants*~tenant.space.usage.bytes      | B     |
| *tenants*~tenant.space.free.bytes       | B     |
| *tenants*~tenant.space.usage.percentage | %     |

</TabItem>
<TabItem value="Volumes" label="Volumes">

| Name                                    | Unit  |
|:----------------------------------------|:------|
| volume-status                           | N/A   |
| *volumes*~volume.space.usage.bytes      | B     |
| *volumes*~volume.space.free.bytes       | B     |
| *volumes*~volume.space.usage.percentage | %     |

</TabItem>
</Tabs>

## Prerequisites

> ###SNMP Configuration
> The SNMP service must be configured and activated on the host. 
> Please refer to the [official documentation](https://knowledge.hitachivantara.com/Documents/Storage/Content_Platform/9.0.x/Administering_HCP/System_monitoring/03_Configuring_SNMP) from the constructor/editor. 
> Your resource may require a list of addresses authorized to query it to be set up. 
> Please ensure that the addresses of the Centreon pollers are included in this list.
> 
> ###Network flow
> The target resource must be reachable from the Centreon poller on the UDP/161 SNMP port.

Reviewer: ```suggestion
### SNMP Configuration
The SNMP service must be configured and activated on the host. 
Please refer to the [official documentation](https://knowledge.hitachivantara.com/Documents/Storage/Content_Platform/9.0.x/Administering_HCP/System_monitoring/03_Configuring_SNMP) from the constructor/editor. 
Your resource may require a list of addresses authorized to query it to be set up. 
Please ensure that the addresses of the Centreon pollers are included in this list.

### Network flow
The target resource must be reachable from the Centreon poller on the UDP/161 SNMP port.
```

> ### SNMP Configuration
> The SNMP service must be configured and activated on the host. 
> Please refer to the [official documentation](https://knowledge.hitachivantara.com/Documents/Storage/Content_Platform/9.0.x/Administering_HCP/System_monitoring/03_Configuring_SNMP) from the constructor/editor. 
> Your resource may require a list of addresses authorized to query it to be set up. 
> Please ensure that the addresses of the Centreon pollers are included in this list.
> 
> ### Network flow
> The target resource must be reachable from the Centreon poller on the UDP/161 SNMP port.

Reviewer: ```suggestion
### SNMP Configuration

The SNMP service must be configured and activated on the host. 
Please refer to the [official documentation](https://knowledge.hitachivantara.com/Documents/Storage/Content_Platform/9.0.x/Administering_HCP/System_monitoring/03_Configuring_SNMP) from the constructor/editor. 
Your resource may require a list of addresses authorized to query it to be set up. 
Please ensure that the addresses of the Centreon pollers are included in this list.

### Network flow

The target resource must be reachable from the Centreon poller on the UDP/161 SNMP port.
```

|:----------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **HW-Storage-Hitachi-Hcp-SNMP-custom** host template. |

More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).

#### Service discovery

| Rule name                               | Description                                               |
|:----------------------------------------|:----------------------------------------------------------|
| HW-Storage-Hitachi-Hcp-SNMP-Node-Name   | Discover the disk partitions and monitor space occupation |
| HW-Storage-Hitachi-Hcp-SNMP-Tenant-Name | Discover the disk partitions and monitor space occupation |

> | HW-Storage-Hitachi-Hcp-SNMP-Volume-Name | Discover the disk partitions and monitor space occupation |

Reviewer 2: all identical

> | HW-Storage-Hitachi-Hcp-SNMP-Node-Name   | Discover the disk partitions and monitor space occupation |
> | HW-Storage-Hitachi-Hcp-SNMP-Tenant-Name | Discover the disk partitions and monitor space occupation |
> | HW-Storage-Hitachi-Hcp-SNMP-Volume-Name | Discover the disk partitions and monitor space occupation |

Reviewer: ```suggestion
| HW-Storage-Hitachi-Hcp-SNMP-Node-Name   | Discover nodes and monitor them |
| HW-Storage-Hitachi-Hcp-SNMP-Tenant-Name | Discover tenants and monitor them |
| HW-Storage-Hitachi-Hcp-SNMP-Volume-Name | Discover the disk partitions and monitor space occupation |
```

> To control your Hitachi HCP, the SNMP must be configured.

Reviewer 2: incomplete prereqs

> To control your Hitachi HCP, the SNMP must be configured.
> (https://knowledge.hitachivantara.com/Documents/Storage/Content_Platform/9.0.x/Administering_HCP/System_monitoring/03_Configuring_SNMP)

Reviewer: ```suggestion
###SNMP Configuration
The SNMP service must be configured and activated on the host. 
Please refer to the [official documentation](https://knowledge.hitachivantara.com/Documents/Storage/Content_Platform/9.0.x/Administering_HCP/System_monitoring/03_Configuring_SNMP) from the constructor/editor. 
Your resource may require a list of addresses authorized to query it to be set up. 
Please ensure that the addresses of the Centreon pollers are included in this list.

###Network flow
The target resource must be reachable from the Centreon poller on the UDP/161 SNMP port.
```
